.github/workflows/test.yml

```diff
@@ -153,6 +169,12 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha || github.sha }}
 
+      - name: install-hyperlane-cli
+        id: install-hyperlane-cli
+        uses: ./.github/actions/install-cli
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+
```

Comment:
is this used?

Reply:
yep, we install the cli and then when we run the e2e tests we use the bundled cli instead of relying on the monorepo code